.github/workflows/health-check.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '8.x', '10.x', '12.x', '14.x' ]
+        node: [ '10.13.x', '12.x', '14.x' ]
```

Comment:
This should be left as `10.x`. It definitely shouldn't be `10.13.x` if the min node version in the `package.json` is being bumped to `10.14.2`